crates/orderbook/openapi.yml

```diff
@@ -1192,12 +1192,12 @@ components:
           allOf:
             - $ref: "#/components/schemas/TransactionHash"
           nullable: true
-        feePolicies:
+        executedProtocolFees:
```

Comment:
Is it sufficient to expose this information on the trade? Currently, we are returning the executed fees on the order (and the frontend plans to display the fee on the order page). Are we also planning to return it there?

Reply:
Hm, will ask the frontend. If they want it on order api as well, I think we can add that in a separate PR.